common/ripple/userUtils.py

```diff
@@ -753,46 +553,49 @@ async def compareHWID(userID: int, mac: str, unique: str, disk: str) -> bool:
     )
 
 
-async def logHardware(userID: int, hashes: list[str], activation: bool = False) -> bool:
+def validate_hwid_set(hwid_set: list[str]) -> bool:
+    """Validate that the mac addresses, unique id, and disk id are present."""
+    return all(hwid_set[2:5])
+
+
+async def associate_user_with_hwids_and_restrict_if_multiaccounting(

```

Comment:
Some of these funcs still have _way_ too many responsibilities. I've stripped a couple (e.g. this was previously _also_ verifying the hwid data was present, which I separated out). To be refactored further.

Reply:
`logHardware` -> (`validate_hwid_set` and `associate_user_with_hwids_and_restrict_if_multiaccounting`) lol